src/index.ts

```diff
@@ -224,6 +224,28 @@ function b32decodeXemAddr(data: string): Buffer {
   return nemSdk.default.utils.convert.ua2hex(nemSdk.default.model.address.b32decode(address));
 }
 
+function ontHexToBase58(hexEncoded: Buffer): string {
+  let addr: Buffer;
+  const ADDR_VERSION = '17';
+  if(hexEncoded.toString('hex').length !== 40) {
+    throw Error('Unrecognised address format');
+  }
+  addr = Buffer.concat([new Buffer(ADDR_VERSION,'hex'), hexEncoded]);
+  return bs58check.encode(addr);
+}
+
+function ontBase58ToHex(base58Encoded: string): Buffer {
+  let hexEncoded: Buffer;
+  if(base58Encoded.length !== 34) {
+    throw Error('Unrecognised address format');
+  }
+  hexEncoded = bs58check.decode(base58Encoded).slice(1,40);
```

Comment:
The value here is not hex encoded, and it's only 20 bytes long. If you want to slice off the first byte, you should call `.slice(1)` - but first you should check the version byte is the expected one.

Reply:
Done